actions/restore.py

```diff
@@ -90,13 +91,13 @@ def unpack_resource():
 
 
 def is_v3_backup():
-    ''' See if the backup file contains a db file indicating a v3 backup '''
-    cmd = "tar -tvf {0} --wildcards '*/db'".format(SNAPSHOT_ARCHIVE)
+    ''' See if the backup file does not contain a wal file indicating a v3 backup '''
+    cmd = "tar -tvf {0} --wildcards '*/wal'".format(SNAPSHOT_ARCHIVE)
     try:
         check_call(split(cmd))
     except CalledProcessError:
-        return False
-    return True
+        return True
+    return False
```

Comment:
i wish i had more context here as to why there's a new command and flipping return values.  Is it because only v3 fails on `/wal` so 

if this command succeeds:
* v1 or v2
if it fails:
* v3

Reply:
I'm not exactly sure but I think `tar -tvf {0} --wildcards '*/db'` command returned successfully for both v1/v2 and v3 since both contained a `db` file. There were 2 approaches we could take to determine the version, which is either checking the directory structure(db file is at root of the archive for v3, for v2 it is in some other folder) or checking if a `wal` file exists. I've went ahead with the `wal` option, to me which seems more solid and future proof. Will add a comment describing the approach. We can go with something else if there are better suggestions.